### PR TITLE
Added screenshot of 6.1 configuration panel for description file

### PR DESCRIPTION
**Pull request by fvigo**
## Status
Ready

## Description
Added screenshot of 6.1 configuration panel for description file

### CODE REVIEW

File: docs/documentation/integration-description.md [hunk 1]
```diff
@@ -42,10 +42,17 @@ This is the contents of the `HelloWorld_description.md` file:
...
-The content of the description file will be displayed 
+The content of the description file will be displayed:
 
 ![](../doc_imgs/integrations/description.md_example.png)
+
+### Cortex XSOAR versions 6.1 and above
+
+Starting from version 6.1, the content of the description file is shown on the side of the configuration data:
+
+![](../doc_imgs/integrations/integration-config-panel-61.png)
```
Comment (ShahafBenYakir) [inline docs/documentation/integration-description.md:58]: This image looks the same as the previous one for me. 
I think it should look like this:
image

File: docs/documentation/integration-description.md [hunk 2]
```diff
@@ -42,10 +42,17 @@ This is the contents of the `HelloWorld_description.md` file:
 - You can use the following API Key: `43ea9b2d-4998-43a6-ae91-aba62a26868c`
 ```
 
-To display the integration description, click on the question mark button in the integration configuration panel
+### Cortex XSOAR versions up to 6.0
```
Comment (ShahafBenYakir) [inline docs/documentation/integration-description.md:45]:
```suggestion
### Cortex XSOAR versions up to 6.0 (not including)
```

File: docs/documentation/integration-description.md [hunk 3]
```diff
@@ -42,10 +42,17 @@ This is the contents of the `HelloWorld_description.md` file:
...
+To display the integration description in versions of XSOAR up to 6.0, click on the question mark button in the integration configuration panel:
 
 ![](../doc_imgs/integrations/description_question_mark.png)
 
-The content of the description file will be displayed 
+The content of the description file will be displayed:
 
 ![](../doc_imgs/integrations/description.md_example.png)
+
+### Cortex XSOAR versions 6.1 and above
```
Comment (ShahafBenYakir) [inline docs/documentation/integration-description.md:54]:
```suggestion
### Cortex XSOAR versions 6.0 and above (including)
```

File: docs/documentation/integration-description.md (hunk 1 above)
Comment (fvigo) [reply docs/documentation/integration-description.md:58]: done

File: docs/documentation/integration-description.md (hunk 2 above)
Comment (fvigo) [reply docs/documentation/integration-description.md:45]: I kept 6.0 and added "including" in the text

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**